tests/lib/uicc_lwm2m/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+tests:
+  uicc_lwm2m.unit_test:
+    sysbuild: true
+    tags: uicc_lwm2m sysbuild
```

Comment:
@nordic-piks could you check these tags?

Reply:
There is needed `ci_samples_cellular`

Reply:
No other `testcase.yaml` file have the `ci_samples_cellular` tag. Is this something new?

Reply:
Indeed, I overlooked that this is new test not another sample.
Tag is needed at samples/cellular/uicc_lwm2m/sample.yaml
At tests/lib/uicc_lwm2m/testcase.yaml there should be new tag added e.g. ci_tests_lib_uicc_lwm2m, Plus new tag needs to be added at https://github.com/nrfconnect/sdk-nrf/blob/main/scripts/ci/tags.yaml + dependencies for this specific test.

Reply:
Right, I'll add a new tag for this tests.